dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/export/event/DefaultEventChangeLogService.java

```diff
@@ -57,6 +66,45 @@ public Page<EventChangeLog> getEventChangeLog(
         eventUid, operationParams.getOrder(), pageParams);
   }
 
+  @Override
+  @Transactional(readOnly = true)
+  public List<TrackedEntityDataValueChangeLog> getTrackedEntityDataValueChangeLogs(
+      TrackedEntityDataValueChangeLogQueryParams params) {
```

Comment:
TrackedEntityDataValueChangeLogQueryParams says 

https://github.com/dhis2/dhis2-core/blob/34da9f9b0e1160dc7d8800ba3e0d7d6c5be25d4b/dhis-2/dhis-api/src/main/java/org/hisp/dhis/trackedentity/TrackedEntityDataValueChangeLogQueryParams.java#L44

is it audit or changelog? 😅

Reply:
This is change log, I'll remove that comment.